src/commands.ts

```diff
@@ -10,6 +10,7 @@ import { CmdJson } from "src/schema/schema.cmd"
 import { CommandRunner } from "src/shell"
 import { CommandConfigs } from "src/types"
 
+export const PIPELINE_SCRIPTS_REF = "PIPELINE_SCRIPTS_REF"
```

Comment:
What does this do?

Reply:
just having a constant (for autocomplete) vs hardcoding as string